jsonschema/validators.py

```diff
@@ -63,12 +68,30 @@ def _id_of(schema):
     return schema.get(u"$id", u"")
 
 
+def _store_schema_list():
+    return [
+        (id, validator.META_SCHEMA) for id, validator in meta_schemas.items()
+    ] + [
+        (id, schema) for id, schema in vocabulary_schemas.items()
+    ]
+
+
+def _applicable_validators(schema):
+    """
+    Returns a list of validators that should apply for the given schema
+    Used for draft2020-12
+    """
+    return schema.items()
+
+
 def create(
     meta_schema,
+    vocabulary_schemas=(),
     validators=(),
     version=None,
     type_checker=_types.draft7_type_checker,
     id_of=_id_of,
+    applicable_validators=_applicable_validators
```

Comment:
It's short enough to just inline this.

```suggestion
    applicable_validators=lambda schema: schema.items(),
```

Reply:
I agree, code is updated